tests/phpunit-mysql8.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Use this file to start a MySQL8 database using Docker and then run the test suite on the MySQL8 database.
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR
+cd ..
+
+docker run --rm --name mysql8-magic-query-test -p 3306:3306 -p 33060:33060 -e MYSQL_ROOT_PASSWORD= -e MYSQL_ALLOW_EMPTY_PASSWORD=1 -d mysql:8 mysqld --default-authentication-plugin=mysql_native_password
```

Comment:
This command may fail if ports are already taken. Do we want to handle this ?

Reply:
No, it's just a helper to run tests easily without having to install mysql